frontend/src/common/analytics/events.ts

```diff
@@ -31,6 +31,7 @@ export enum EVENTS {
   WMG_CLICK_NAV = "WMG_CLICK_NAV",
   WMG_HEATMAP_LOADED = "WMG_HEATMAP_LOADED",
   WMG_CLICKED = "WMG_CLICKED",
+  WMG_CITATION_CLICKED = "WMG_CITATION_CLICKED",
```

Comment:
I'm going to need access to Plausible and also waiting on Amanda to provide some additional info - this shouldn't be barring though since it's a separate ticket

Reply:
ok will reach out for it! thanks 🙏🏽